src/d2l-rubric.js

```diff
@@ -535,10 +539,16 @@ const component = {
 
 	attached: function() {
 		window.addEventListener('beforeunload', (e) => this._showPopoutDialogWarning(e));
+		window.addEventListener('cell-assessment-map-update', (e) => this.updateAssessmentMap(e));
```

Comment:
nit: you could do this in one line:
```suggestion
		window.addEventListener('cell-assessment-map-update', (e) => this._criterionAssessmentMap = e.detail.value);
```